client/src/components/profile/components/certifications.tsx

```diff
@@ -69,15 +70,17 @@ function Certificates({
 }: CertificationProps): JSX.Element {
   const { t } = useTranslation();
   return (
-    <FullWidthRow className='certifications'>
-      <h2>{t('profile.fcc-certs')}</h2>
+    <FullWidthRow className='profile-certifications'>
+      <h2 id='fcc-certifications'>{t('profile.fcc-certs')}</h2>
```

Comment:
i think, it can be better to use css class here (and not id)

Reply:
this id is not used anywhere. Removed it.